account_tax_one_vat_sale/models/sale_order_line.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class SaleOrderLine(models.Model):
+    _name = "sale.order.line"
+    _inherit = ["sale.order.line", "one.vat.mixin"]
+
+    @api.constrains("tax_id")
+    def _check_only_one_vat(self):
+        self._check_only_one_vat_tax_field("tax_id")
+
+    @api.onchange("tax_id")
+    def _onchange_tax_id(self):
```

Comment:
To avoid conflicts, name the method with what it is doing, not what is triggering it

```suggestion
    def _onchange_only_one_vat(self):
```